config/action.d/firewallcmd-allports.conf

```diff
@@ -18,21 +11,19 @@ actionstop = firewall-cmd --direct --remove-rule ipv4 filter <chain> 0 -j f2b-<n
              firewall-cmd --direct --remove-rules ipv4 filter f2b-<name>
              firewall-cmd --direct --remove-chain ipv4 filter f2b-<name>
 
-
 # Example actioncheck: firewall-cmd --direct --get-chains ipv4 filter | sed -e 's, ,\n,g' | grep -q '^f2b-recidive$'
 
 actioncheck = firewall-cmd --direct --get-chains ipv4 filter | sed -e 's, ,\n,g' | grep -q '^f2b-<name>$'
 
-actionban = firewall-cmd --direct --add-rule ipv4 filter f2b-<name> 0 -s <ip> -j <blocktype>
+actionban = firewall-cmd --direct --add-rule ipv4 filter f2b-<name> 0 -s <ip> -j REJECT
```

Comment:
> and also fixed firewallcmd allports and multiport.

??? What was fixed here?

Block type can also be specified in Init block as default, and can be rewritten in jail or in includes (if expected):
```
[Init]
blocktype = REJECT
```

BTW, I do not understand why following test does not fail by this action (https://github.com/fail2ban/fail2ban/blob/5d0d96a5cbe7b8b45173f440568386ecd7d1924b/fail2ban/tests/clientreadertestcase.py#L652)